pageserver/src/tenant/storage_layer/delta_layer.rs

```diff
@@ -963,14 +964,25 @@ impl DeltaLayerInner {
                 .blobs_at
                 .as_slice()
                 .iter()
-                .map(|(_, blob_meta)| format!("{}@{}", blob_meta.key, blob_meta.lsn))
+                .filter_map(|(_, blob_meta)| {
+                    if blob_meta.key.is_rel_dir_key() || blob_meta.key == DBDIR_KEY {
```

Comment:
This condition while only interesting for logging, maybe we could move it in as a `Key` method to avoid duplication? `is_oversized_nagged` or `_warned` or something like that?

Reply:
For me, the fact that the condition is only interesting for logging means that it shouldn't be part of the `Key` api.
I'll punt, but if you feel strongly lmk and I'll lift it.